src/drive/web/modules/views/Recent/index.jsx

```diff
@@ -55,11 +54,11 @@ export const RecentView = ({ router, children }) => {
     dispatch,
     router,
     location,
-    hasWriteAccess: false,
+    hasWriteAccess: true,
     canMove: true
   }
   const actions = useActions(
-    [share, download, trash, open, rename, move, qualify, versions, offline],
+    [download, trash, open, rename, move, qualify, versions, offline],
```

Comment:
we can't share anymore on the recent view?

Reply:
It was never an option.